update_npm_dependencies.rb

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env ruby
+
+require 'optparse'
+require 'json'
+
+def package_name(npm_module_name)
+  "nodejs-#{npm_module_name}"
+end
+
+def package_dir(npm_module_name)
+  "packages/foreman/#{package_name(npm_module_name)}"
+end
+
+def package_exist?(npm_module_name)
+  File.directory?(package_dir(npm_module_name))
+end
+
+def current_version(npm_module_name)
+  if package_exist?(npm_module_name)
+    spec_file = "#{package_dir(npm_module_name)}/#{package_name(npm_module_name)}.spec"
+    version_line = File.readlines(spec_file).grep(/^\W*Version:.*/).first
+    version_line.gsub(/^\W*Version:\W*/, '').strip
+  end
+end
+
+def bundle_strategy(npm_module_name)
+  if package_exist?(npm_module_name)
+    # new packages default to bundle
```

Comment:
I prefer to not default to bundles. Only if there are too many deps that aren't present or if it would result in conflicting situations where package x requires a version package y can't deal with. You can also leave it empty and let `add_npm_package.sh` deal with this. It queries `api.npms.io` to get the number of dependencies.

Reply:
Ok, I'll probably leave it empty for now and we can update it later.